internal/spec/parser.go

```diff
@@ -53,6 +53,11 @@ func (p *Parser) Decode(spec string) (interface{}, error) {
 		return &kio.ByteReader{Reader: p.Reader}, nil
 	}
 
+	// Inline resources
+	if strings.ContainsRune(spec, '\n') {
```

Comment:
I think this works for now, but might be something we want to revisit later on given how many nuances there may be